apple-pie/styles/style.css

```
@@ -0,0 +1,95 @@
+.container{
```

El espacio en blanco que tiene tu pagina lo podemos quitar aplicando margin y padding 0 al body a traves de la etiqueta body
```suggestion
body {
    margin: 0;
    padding: 0;
}
```